pkg/tools/variables.go

```diff
@@ -51,3 +53,53 @@ func ListVariables(client apiClient.ClientInterface) (tool mcp.Tool, handler ser
 			return mcp.NewToolResultText(string(variablesJSON)), nil
 		}
 }
+
+func CreateProjectVariable(client apiClient.ClientInterface) (tool mcp.Tool, handler server.ToolHandlerFunc) {
```

Comment:
There are two types of variables (List and Text) in Perses. You are creating a text variable. 
So, could you please change the variable name to `CreateProjectTextVariable`?